### DISCUSSION TITLE
fr/models/yolo11/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#18584] [Docs]
# fr/models/yolo11/

Découvrez YOLO11, la dernière avancée en matière de détection d'objets de pointe, qui offre une précision et une efficacité inégalées pour diverses tâches de vision par ordinateur.

https://docs.ultralytics.com/fr/models/yolo11/

**Comment by abaldanz** [comment]
Hi, team. I would like to train a yolov11 from scratch on the COCO database (the idea is to have a pre-train model on imgsz=1280). I just wanted to know what parameters you usually use to train models from scratch on large databases like COCO?

Thank you very much!

**Reply by glenn-jocher** [reply]
@abaldanz to train YOLO11 from scratch on the COCO dataset with `imgsz=1280`, you can use the `train` mode with a configuration like `yolo train model=yolo11n.yaml data=coco.yaml epochs=300 imgsz=1280`. Adjust hyperparameters like `batch_size`, `learning_rate`, and `optimizer` based on your hardware and dataset. For detailed guidance, see the [Train documentation](https://docs.ultralytics.com/modes/train/).

**Reply by abaldanz** [reply]
Thanks! I read in a thread on your GitHub repo that you might be releasing a train version of yolov11x on 1280 images. Do you have an approximate date for this ?

**Reply by glenn-jocher** [reply]
We don't have an official release date for YOLOv11x-1280 models yet. For updates, please watch our GitHub repository where new releases are always announced first: https://github.com/ultralytics/ultralytics.

**Comment by UltralyticsAssistant** [comment]
👋 Hello, thank you for your interest in Ultralytics and YOLO11 🚀! We recommend checking out the [Ultralytics Docs](https://docs.ultralytics.com) to get started, where you'll find detailed information on [models like YOLO11](https://docs.ultralytics.com/models/yolo11), as well as many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples.

If you are raising a 🐛 Bug Report or have encountered specific issues, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us understand and debug the problem effectively.

For ❓ Questions, especially regarding custom training, make sure to include comprehensive details such as dataset image examples, training logs, and verification that you're following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community and connect with others! For real-time conversations, visit us on [Discord](https://discord.com/invite/ultralytics) 🎧. For detailed discussions, you can head over to our [Discourse](https://community.ultralytics.com). Prefer Reddit-style threads? Explore ideas on our [Subreddit](https://reddit.com/r/Ultralytics).

## Upgrade

Ensure you're running the latest version of the `ultralytics` package by upgrading in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLO models, including YOLO11, can be seamlessly run in the following verified environments with all required dependencies installed:

- **Notebooks** (free GPU access): <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM: See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI: See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

The CI badge above indicates the status of automated tests that run across macOS, Windows, and Ubuntu daily. For detailed performance insights, you can check our [modes](https://docs.ultralytics.com/modes/) and [tasks](https://docs.ultralytics.com/tasks/).

This is an automated response to guide you in the right direction, but don't worry 😊—an Ultralytics engineer will follow up soon to assist further!
